views/index.pug

```diff
@@ -38,12 +38,25 @@ block content
                 button(class='recipe-favorited button', id=`favoriterecipe:${recipe.id}.cupboard:${user.Cupboards[0].id}`, type='submit') Unfavorite
               else
                 button(class='recipe-favorited button', id=`favoriterecipe:${recipe.id}.cupboard:${user.Cupboards[0].id}`, type='submit') Favorite
-              div(class='rating')
-                span ☆
-                span ☆
-                span ☆
-                span ☆
-                span ☆
+              if recipe.status.starRating
+                div
+                  - let n = 5
+                  div(class='rating')
+                    span(class='rating__button rating__delete fa fa-minus-circle', id=`recipe:${recipe.id}`)
+                    while n > recipe.status.starRating
+                      span(class='rating__button fa fa-star', id=`recipe:${recipe.id}.star:${n}`)
+                      - n--
+                    - let m = recipe.status.starRating
+                      while m >= 1
+                        span(class='rating__button fa fa-star checked', id=`recipe:${recipe.id}.star:${m}`)
+                        - m--
+              else
+                div(class='rating')
+                  span(class='rating__button fa fa-star', id=`recipe:${recipe.id}.star:5`)
```

Comment:
Repeated code! Try looping to create each of these spans instead of rewriting the same code:
```
each val in [1, 2, 3, 4, 5]
    span...
```